match/match.go

```diff
@@ -94,8 +94,14 @@ func matchCreateHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if !userOneValid || !userTwoValid {
-		errMsg := utils.CreateErrorJSON(fmt.Sprintf("Unknown Users: %d | %d", userOne, userTwo))
+	if !userOneValid {
+		errMsg := utils.CreateErrorJSON(fmt.Sprintf("Unknown User: %d", userOne))
```

Comment:
Turning the question around, is this the error message we want when both are wrong? I guess it’s probably fine

Reply:
I think it should fail on the first one, so we don't make a redundant `checkUser` call for `userTwo` when `userOne` is invalid anyway. See my comment below about shifting the calls and checks around.